stable/cert-manager-setup/templates/v1upgrade.yaml

```diff
@@ -0,0 +1,124 @@
+{{- if .Values.installCertManager }}
```

Comment:
Just floating a previous comment back up here so it's not lost - to add a `{{- if (.Capabilities.APIVersions.Has "certmanager.k8s.io/v1alpha1") -}}` check to only run these backup and restore jobs if we're running the old version of certmanager

Reply:
done